component/avatar/avatar.css

```diff
@@ -0,0 +1,37 @@
+/* General */
+
+a {
+    text-decoration: none;
+}
+
+/* Typography */
+
+:root {
+    --font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
+    Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
+/* Avatar */
+
+.avatar {
+    border-radius: 50%;
+    width: 32px;
+    height: 32px;
```

Comment:
Can convert values > 10px to rem.

Reply:
thanks have done the changes in all the avatar classes